lib/rubocop/cop/rails/hash_enum.rb

```diff
@@ -0,0 +1,34 @@
+# frozen_string_literal: true
+
+module RuboCop
+  module Cop
+    module Rails
+      # This cop looks for enums written with array syntax
+      #
+      # @example
+      #   # bad
+      #   enum status: %i(active archived)
+      #
+      #   # bad
+      #   enum status: [:active, :archived]
```

Comment:
I don't think there's any need for both these 'bad' examples since only the syntax differs.

(I'd be inclined to pick the second since it's most common).